### DISCUSSION TITLE
How to enable right side bar on mobile

### DISCUSSION THREAD

**Discussion by DowneyFlyfan** [CaiJimmy/hugo-theme-stack#730] [Q&A]
I can see the right sidebar on my PC by zooming. But that didn't work on my mobile phone and my pad.

**Comment by TigGrak** [comment]
The right side bar may not be displayed to the small screen device so that it can be adaptive to your mobile device

you can click ```menu``` to find the widgets which in right side bar

like this
![1669720436697](https://user-images.githubusercontent.com/85205528/204514850-c382023d-c154-4630-b7fe-f1965945b30d.png)

**Reply by DowneyFlyfan** [reply]
Got it, thank you.

**Comment by CaiJimmy** [comment]
The right sidebar is hidden on smaller screens, there's no option to enable it.

**Reply by DowneyFlyfan** [reply]
Alright, thank you
